doc/whatsnew/fragments/9881.bugfix

```diff
@@ -0,0 +1,3 @@
+Remove `too-few-function-args` in favor of the incumbent message `no-value-for-parameter`.
```

Comment:
Let's make the changelog make sense by modifying the existing entry and hide the churn. I'll add the skipnews label